### ISSUE TITLE
[android] [bug] two pointers tap not disabled when zoomGesturesEnabled = false

### ISSUE THREAD

**Issue by mayardb** [mapbox/mapbox-gl-native#4517]
Hello Mapbox team!

I am playing with the `SupportMapFragmentActivity`. It is a really great feature!

Unfortunately I am able to unzoom whereas the gestures are disabled:
```
    MapboxMapOptions options = new MapboxMapOptions();
    ...
    
    options.scrollGesturesEnabled(false);
    options.zoomGesturesEnabled(false);
    options.tiltGesturesEnabled(false);
    options.rotateGesturesEnabled(false);
```

This happens when taping with two fingers. I fix it by checking if zoom gestures are enabled in the `onTouchEvent`:
```
case MotionEvent.ACTION_POINTER_DOWN:
    // Second pointer down
    mTwoTap = event.getPointerCount() == 2
                    && !mDestroyed
                    && mMapboxMap.getUiSettings().isZoomGesturesEnabled();
```

(I copy the `mDestroy || mMapboxMap.getUiSettings().isZoomGesturesEnabled()` logic from the other gesture checks)

Pull Request following.

**Comment by tobrun** [comment]
@mayardb thanks again for reaching out and reporting a problem!

**Comment by mayardb** [comment]
PullRequest done: #4518.

@tobrun this is a pleasure to contribute. Your improvements are amazing.

**Comment by tobrun** [comment]
Landed in #4550 with b347451
